templates/organization/edit-composition.html

```diff
@@ -43,14 +43,14 @@ <h3 id="relationships">
         <tr>
             <td colspan="7">
                 <a href="{% url 'create-organization-composition' organization.uuid %}">
-                <i class="fa fa-plus"> </i> Add composition
+                <i class="fa fa-plus"> </i> {% trans "Add composition" %}
```

Comment:
I don't completely understand why GitHub is showing this diff, it looks like this change has already been made on `master`:

https://github.com/security-force-monitor/sfm-cms/blob/d549fd8e629d73a5453bd5c9cc34a3c09a1af52b/templates/organization/edit-composition.html#L46

I don't think there's anything you need to do to fix it, I'm just confused about why it thinks this is a new change.

Reply:
Looking below at e.g. [line 130](https://github.com/security-force-monitor/sfm-cms/pull/594/files#diff-fe6f3022773962eee50d4effb1c2bc51R130), I'm wondering if maybe commits from previous PRs are being rendered in this diff. Did you make these changes or are they indeed coming from previous commits? Perfectly fine if they're coming from previous commits, I just want to know which diffs are real.

Reply:
i didn't make these changes!

Reply:
Got it, I'll resolve this thread then.